Pokemon Tabletop Adventure/Data Models/WBPokémon.swift

```diff
@@ -5,7 +5,7 @@
 //  Created by Ian Merry on 08/07/2018.
 //  Copyright © 2018 WeatherBear. All rights reserved.
 //
-
+// swiftlint:disable redundant_string_enum_value
```

Comment:
Already disabling it 🤣

Reply:
Probably... I should just put this one into the disabled rules in the config?